CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ### Fixed
 - **cf-typography:** [PATCH] Fixed old variables removed from cf-core
+- **cf-forms:** [PATCH] Fixed a layout bug in btn-inside-input
```

Comment:
My plan was to merge all these to a single major release when the time comes, but to track changes, however minor, that are happening now so we know where we're at without having to dig through merged PRs.